bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -1009,7 +1025,16 @@ private void updatePanelColors() {
     void parsePanelData(NanoleafControllerConfig config, ContentResponse panelData) {
         // panelData is in format (numPanels, (PanelId, 1, R, G, B, W, TransitionTime) * numPanel)
         @Nullable
-        Write response = gson.fromJson(panelData.getContentAsString(), Write.class);
+        Write response = null;
+
+        String panelDataContent = panelData.getContentAsString();
+        try {
+            response = gson.fromJson(panelDataContent, Write.class);
+        } catch (JsonSyntaxException jse) {
+            logger.error("Unable to parse panel data information from Nanoleaf", jse);
```

Comment:
Unless this will stop the binding from working, log level should be reduced to at least warning.